content/_code-samples/checks/py/account_checks.py

```diff
@@ -0,0 +1,111 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountObjects
+from xrpl.utils import drops_to_xrp, ripple_time_to_datetime
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to the testnetwork
+
+
+
+"""Helper methods for working with token names"""
+
+def hex_to_symbol(hex: str = None) -> str:
```

Comment:
can you use this function instead? 
https://xrpl-py.readthedocs.io/en/stable/source/xrpl.utils.html#xrpl.utils.hex_to_str